pkg/build/controller/strategy/custom_test.go

```diff
@@ -116,6 +116,7 @@ func mockCustomBuild() *buildapi.Build {
 						{Name: "FOO", Value: "BAR"},
 					},
 					ExposeDockerSocket: true,
+					ForcePull: true,
```

Comment:
OK, that helps re: exposeDockerSocket and a "mount" of a docker socket.
And certainly the container ImagePullPolicy and its relationship with the strategy's force pull is central to this change and I just missed that one.

I'll get started ...